spec/requests/documents_spec.rb

```diff
@@ -3,9 +3,9 @@
 RSpec.describe "Documents" do
```

Comment:
We should move some of these around to match the controllers. So we should have a "Content" request test for those checks and delete_draft ones should be moved to Editions request tests. We'll also have to update the paths in the describe methods.

Reply:
All sorted after in person conversation with you :)